m4/acinclude.m4

```diff
@@ -0,0 +1,537 @@
+dnl ##
+dnl ##  acinclude.m4 -- manually provided local Autoconf macros
+dnl ##
+
+dnl ##
+dnl ##  NAME:
+dnl ##    AC_MSG_TITLE -- Display a configuration title
+dnl ##
+dnl ##  USAGE:
+dnl ##    AC_MSG_TITLE(<name>, <version>)
+dnl ##
+
+AC_DEFUN([AC_MSG_TITLE],[
+    _AS_ECHO([Configuring $1, Version $2])
+])
+
+dnl ##
+dnl ##  NAME:
+dnl ##    AC_MSG_HEADER -- Display a configuration header
+dnl ##
+dnl ##  USAGE:
+dnl ##    AC_MSG_HEADER(<text>)
+dnl ##
+
+AC_DEFUN([AC_MSG_HEADER],[
+    _AS_ECHO([])
+    _AS_ECHO([=== $1 ===])
+])
+
+dnl ##
+dnl ##  NAME:
+dnl ##    AC_MSG_VERBOSE -- Display a message under --verbose
+dnl ##
+dnl ##  USAGE:
+dnl ##    AC_MSG_VERBOSE(<text>)
+dnl ##
+
+AC_DEFUN([AC_MSG_VERBOSE], [
+    if test ".$verbose" = .yes; then
+        _AS_ECHO([$1])
+    fi
+])
+
+dnl ##
+dnl ##  NAME:
+dnl ##    RPM_CHECK_LIB -- Check for third-party libraries
```

Comment:
and such things as well... personally I think we should just stick to pkgconfig and do not care about anything else. pkgconfig is standard de-facto.